packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop           | Type                                                               | Default     | Description                                                                    | Required |
+| -------------- | ------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children       | `string`                                                           | `undefined` | The value to be rendered within the component                                  | ✅       |
```

Comment:
With amount, we're dealing with numbers. I think the value being passed should be a number otherwise it'll involve internal parsing of string to number. Instead we can have a `value` prop for accepting the numeric literal.